app/classification.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const natural = require('natural');
+const wordnet = new natural.WordNet();
+
+var teacherArray = ['taught','coached', 'trained', 'educated'];
+var studentArray = ['learned from'];
+var wroteArray = ['wrote','created', 'composed'];
+var playedArray = ['played','performed'];
```

Comment:
these lines should be removed as they are properly defined with `const` afterwards.

(this is also the reason why codacy complains)